src/tir/analysis/verify_gpu_code.cc

```diff
@@ -134,25 +138,28 @@ class GPUCodeVerifier : public StmtExprVisitor {
     }
   }
 
+  void VisitStmt_(const ForNode* op) {
+    if (op->loop_var->name_hint == "vthread.s") {
```

Comment:
Since there's a check in `AttrStmtNode`, is it necessary to further check `ForNode`?

Reply:
@jcf94  It is necessary. The pragma will be translated to a for loop in the InjectiveVirturalThread pass. Adding this check makes the verification work both before and after InjectVirtualThread pass.